api/controllers/Cid_EntryController.js

```diff
@@ -71,6 +71,12 @@ module.exports = {
       .populate('female_active_movements_in')
       .populate('female_active_movements_out')
       .then(centres => _.map(centres, centre => Centres.publishUpdate(centre.id, centre)))
+      .then(() => {
```

Comment:
You are of course correct. The integration tests break when I return Centres.update and not Movement. I need to solve that and I'm golden